src/main/java/seedu/recipe/ui/RecipeDetailsPanel.java

```diff
@@ -62,13 +62,21 @@ public void setRecipeDetails(Recipe recipe) {
                     });
             steps.getChildren().clear();
             stepsTitle.setText("Instructions");
+            /*
             recipe.getSteps().stream()
                     .forEach(step -> {
                         String eachStep = recipe.getSteps().indexOf(step) + 1 + ".  " + step.step;
                         Label stepLabel = new Label(eachStep);
                         stepLabel.setWrapText(true);
                         steps.getChildren().add(stepLabel);
                     });
+             */
+            for (int i = 1; i <= recipe.getSteps().size(); ++i) {
+                String eachStep = i + ".  " + recipe.getSteps().get(i - 1).step;
+                Label stepLabel = new Label(eachStep);
+                stepLabel.setWrapText(true);
```

Comment:
Perhaps you could wrap the text of the description as well when a long description is used?